safe_cli/operators/safe_operator.py

```diff
@@ -9,6 +9,9 @@
 from eth_typing import ChecksumAddress
 from eth_utils import ValidationError
 from hexbytes import HexBytes
+from ledgereth import get_account_by_path
```

Comment:
Please refactor this in another class. Also remember that importing `ledgereth` must be optional for Safe-CLI users. When calling `load_ledger_cli_owners` I would expect that job to be done in another class, getting a warning if ledger is not connected/detected or getting a warning about `ledgereth` to be installed to get it working (if dependencies are not installed)